content/en/docs/configuration/acme/dns01/route53.md

```diff
@@ -38,7 +38,10 @@ permissions:
     },
     {
       "Effect": "Allow",
-      "Action": "route53:ListHostedZonesByName",
+      "Action": [
+        "route53:ListHostedZonesByName", 
+        "route53:ListHostedZones"
+      ],
```

Comment:
I don't know enough about AWS permissions to know whether this is safe. 

I see others have mentioned this permission in cert-manager issues:
 * https://github.com/search?q=org%3Ajetstack+ListHostedZones&type=issues

But I'm not sure we need both `ListHostedZonesByName` **and** `ListHostedZones`.

Please explain why `ListHostedZonesByName` is insufficient and update the `Note: ` below.